src/main/java/frc/team166/robot/subsystems/Lift.java

```diff
@@ -215,6 +226,49 @@ protected void interrupted() {
         };
     }
 
+    public Command LowerLiftToLimitSwitch() {
+        return new SubsystemCommand(this) {
+            @Override
+            protected void initialize() {
+                disengageBrake();
+            }
+
+            @Override
+            protected void execute() {
+                lowerLift();
+            }
+
+            @Override
+            protected boolean isFinished() {
+                if (bottomLimitSwitch.get()) {
+                    return true;
+                } else {
+                    return false;
+                }
+            }
+        };
+    }
+
+    public Command LowerLiftForTime() {
+        return new SubsystemCommand(this) {
+            @Override
+            protected void initialize() {
+                disengageBrake();
+                setTimeout(2);
+            }
+
+            @Override
+            protected void execute() {
+                lowerLift();
+            }
+
+            @Override
+            protected boolean isFinished() {
+                return isTimedOut();
+            }
+        };
```

Comment:
Don't forget to add an end() that calls engageBrake(); and liftDrive.stopMotor(). The command above should also have an end() function.